commonblocks/static/commonblocks/js/hallo-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * registerHalloPlugin() applies to all editor instances of a Page.
+ * We want to encapsulate custom plugin configuration, per field.
+ */
+function makeHalloSimpleRichTextEditable(id, plugins) {
+    var defaultPlugins = $.extend({}, halloPlugins);
```

Comment:
This is only doing a shallow copy. Just to be safe you may want to do `$.extend(true, {}, halloPlugins);` to get a deep copy.